packages/engine/lib/stateful/src/field/key.rs

```diff
@@ -72,15 +77,10 @@ impl RootFieldKey {
         )))
     }
 
-    /// Returns the key as string
+    /// Returns the key as [`String`].
```

Comment:
But it doesn't return it as `String`?

Reply:
Fixed in 0f1b85bc36d0a36fcf5a5ca7cb8ecf0fd5bc90b6